src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
```

Comment:
The `balls` array is declared but not initialized with `Ball` objects. You need to populate this array with `Ball` instances using the `Lottery.getRandomBall()` method before iterating over it.